pallets/bridge-common/src/lib.rs

```diff
@@ -33,6 +33,7 @@
 pub mod beefy_types;
 pub mod bitfield;
 pub mod simplified_mmr_proof;
+pub mod simplified_proof;
```

Comment:
`simplified_proof` and `simplified_mmr_proof` contains the same structure. We can remove `simplified_mmr_proof` module

Reply:
Done